client/src/app/users/user.service.ts

```diff
@@ -119,6 +119,7 @@ export class UserService {
 
   addUser(newUser: Partial<User>): Observable<string> {
     // Send post request to add a new user with the user data as the body.
+    // `res.id` should be the MongoDB ID of the newly added `User`.
```

Comment:
Should we say anything about res being the response? Or, should the name of the thing just be changed to `response` instead of `res` to help clarify?

Reply:
I like the idea of changing the name to `response`.